js/app/historyStore.js

```diff
@@ -389,8 +389,9 @@ var HistoryStore = new Lang.Class({
         if (!item || !item.can_share)
             return;
 
+        let model = item.media_model || item.model;
```

Comment:
later in this method, the metric system is still referring to item.model, instead of model, I guess we want to track the real thing we shared.

Reply:
Good catch, I missed that one